external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopVillainsIT.java

```diff
@@ -0,0 +1,202 @@
+package io.quarkus.ts.external.applications;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.apache.http.HttpStatus;
+import org.hamcrest.core.Is;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import io.quarkus.test.bootstrap.DefaultService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
+import io.quarkus.test.scenarios.annotations.DisabledOnQuarkusVersion;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.GitRepositoryQuarkusApplication;
+import io.restassured.http.ContentType;
+
+@DisabledOnNative
+@DisabledOnQuarkusVersion(version = "9\\..*", reason = "999-SNAPSHOT is not available in the Maven repositories in OpenShift")
```

Comment:
Just out of curiosity, why are we not matching `999-SNAPSHOT` here instead?

Reply:
We could amend the condition to match exactly 999-SNAPSHOT. At the moment, this condition works fine.
As there are other places where this condition is in place, if you want to change the condition, we should do it in a separate task for all of them.